README.md

```diff
@@ -43,15 +43,16 @@ Create a shared secret:
 openssl rand -hex 16
 ```
 
-Download and run the plugin:
+Run the plugin:
 
 ```shell
-docker run -d \
+docker build -t drone-validation .
+
+docker run \
   --publish=3000:3000 \
   --env=DRONE_DEBUG=true \
-  --env=DRONE_SECRET=<your_shared_secret> \
-  --restart=always \
-  --name=<container_name> quay.io/mongodb-labs/drone-validation:latest 
+  --env=DRONE_VALIDATE_PLUGIN_SECRET=$(openssl rand -hex 16) \
```

Comment:
The same secret needs to be configured in drone-server. If we suggest creating a new one dynamically every time the plugin starts, users will have to discover the runtime password by themselves.

Reply:
This makes this a working example to run locally. This would not be a valid way to run in any real world deployment.

Reply:
changes the wording of the readme